src/util/interval_constraint.h

```diff
@@ -0,0 +1,22 @@
+/*******************************************************************\
+
+Module: Interval constraint
+
+Author: Jeannie Moulton
+
+\*******************************************************************/
+
+#ifndef CPROVER_UTIL_INTERVAL_CONSTRAINT_H
+#define CPROVER_UTIL_INTERVAL_CONSTRAINT_H
+
+#include "expr.h"
+#include "integer_interval.h"
```

Comment:
exprt and integer_intervalt can be forward declared

Reply:
done for exprt. cannot be done for integer_interval it seems.